spark/src/test/scala/org/apache/comet/parquet/ParquetReadSuite.scala

```diff
@@ -1460,6 +1460,33 @@ class ParquetReadV1Suite extends ParquetReadSuite with AdaptiveSparkPlanHelper {
           v1 = Some("parquet"))
     }
   }
+
+  test("test V1 parquet scan filter pushdown uses native_iceberg_compat") {
+    withTempPath { path =>
+      val rows = 1000
+      spark.range(rows).toDF("a").write.parquet(path.toString)
```

Comment:
Would you mind covering all other types?

Reply:
> Would you mind covering all other types?

Sure, I will add later

Reply:
added